app/src/main/java/io/apicurio/registry/storage/impl/AbstractMapRegistryStorage.java

```diff
@@ -155,6 +156,15 @@ private boolean valueContainsSearch(String search, String artifactId, String met
         return value != null && StringUtils.containsIgnoreCase(value, search.toLowerCase());
     }
 
+    @Nullable
```

Comment:
Out of curiosity - what does this do?

Reply:
I mean obviously it marks the method as possibly returning null, but what does the IDE do with that info?

Reply:
Auto-added by the IDE since I'm explicitly returning `null`, nothing functionally, just marker afaik.

Reply:
For Idea, [this](https://www.jetbrains.com/help/idea/nullable-and-notnull-annotations.html) is what is doing.

Reply:
Interesting, thanks for the link.